### ISSUE TITLE
[BUG] FPS Micro-stutter and performance

### ISSUE THREAD

**Issue by maxrouter2** [flybywiresim/a32nx#3368]
**Mod Version**
v0.5.2 
 "built": "2021-02-05T16:30:56+00:00",
  "ref": "master",
  "sha": "aff960bf74a0a806cb56ae8a14d5990a6181dff9",
  "actor": "Benjozork",
  "event_name": "manual"

**Describe the bug**
i know this is a known issue but i was wondering if there is any more help on how to reduce this fps and microstuttering issue. thank you to the developers for even making this mod possible. the defualt a320 is unplyable with the bugs and issues. this mod has not only fixed everything but more features added which is amazing. however, every few seconds in the game the frame lags and worse during landing and taking off. i have turnt off the FO screens and set cockpit refresh rate to medium. i have even tried setting all settings to the lowest possible (off or low), but i still ecnounter the issues and havent seen a topic made since last year. any help would be apprecaited. thankyou! also i havent got the best specs for my gaming laptop but i dont think i should encounter the performance that i do. 

**To Reproduce**
1. Set up flight anywhere
2. FO screens off
3. observe behavouir 

**Expected behavior**
Smooth and no stutter during flight (maybe the occasional one)

**Actual behavior**
every few seconds a micro stutter which could pause between a mili second or a couple of seconds. 

**References**
n/a

**Additional context**
i5-9300H 2.4ghz oc to 4.10ghz
gtx 1050
8gb ram

Is this a problem in the vanilla unmodded game?
not sure

**Comment by Benjozork** [comment]
The development version contains performance optimizations. Have you tried it ?

**Comment by maxrouter2** [comment]
I am using the fbw installer and using stable version. V0.5.2. I have tired the development version and issue persists.(if not ever so slightly worse due to the printer and efb). The strange thing is aswell my fps counter doesn’t seem to drop as the stutter happens. It’s like a latency thing almost. I will try again with the dev version. Thank you for your response.

**Comment by maxrouter2** [comment]
UPDATE: after trying the developers version, the performance seems to be far better. However I did reset and mess around with a few settings. I would recommend removing any interference from Nvidia control panel (resetting settings to default). And disassociating msfs 2020 from nvidida geforce experience. It seemed nvidia was conflicting. I also removed the vfr map and objectives from the custom toolbar in msfs. And I also put my windows in the latest update and in hardware accelerated gpu mode. This seems to have solved the stutter. There are a few spikes on takeoff and landing but this is a working project. But hats off to the team this mod has blown my mind. Nothing payware could’ve done so thankyou!
